### PR TITLE
feat: use `PORT` environment variable to change port

### PR DESCRIPTION
**Pull request by m4rocks**
This can be useful when deploying to other providers.

Also more convenient for projects with multiple developers with port preferences

### CODE REVIEW

**Review by fdemir — APPROVED** [review]

**Review by lucacasonato — COMMENTED** [review]
This does not do what you think it does. It will still listen on port 8000 only, but it will now print that it's listening on a different port.

**Comment by m4rocks** [comment]
damn i'm stupid. i didn't notice it was inside console.log. thanks for pointing out

File: src/server/mod.ts [hunk 1]
```diff
@@ -50,10 +50,21 @@ export interface Manifest {
 
 export { ServerContext };
 
+const defaultPort = parseInt(Deno.env.get("PORT") || "") || 8000;
```
Comment (lino-levan) [inline src/server/mod.ts]: Couldn't this be written more simply as:

```javascript
const defaultPort = parseInt(Deno.env.get("PORT") || "8000");
```

**Comment by iuioiua** [comment]
This seems like a useful addition. Are you able to add a simple test?

File: src/server/mod.ts [hunk 2]
```diff
@@ -60,9 +60,11 @@ export interface DenoConfig {
 
 export { ServerContext };
 
+const port = parseInt(Deno.env.get("PORT") || "8000");
```
Comment (iuioiua) [inline src/server/mod.ts]: Nit: I think it's fine to make this inline.

**Comment by iuioiua** [comment]
Hi @notangelmario, we're happy to merge this once a test is added.

**Comment by m4rocks** [comment]
sure, will get on it asap

**Review by iuioiua — APPROVED** [review]
I've added a test. LGTM now!

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**